extensions/target-specific/stl-helpers/stl-helpers.js

```diff
@@ -106,7 +153,7 @@ DbgObject.AddArrayField(
 
 DbgObject.AddArrayField(
```

Comment:
Is it worth adding "Pairs" to std::map as well?

Reply:
Not sure I understand -- there's code around line 65 that adds Pairs to std::map

Reply:
Missed that - sorry! LGTM